src/image_deid_etl/image_deid_etl/custom_etl.py

```diff
@@ -367,7 +367,7 @@ def convert_dicom2nifti(data_dir):
     for acquisition in acq_list:
         # print(acquisition)
         if glob(acquisition+'/*.nii.gz')==[]: # if there are no niftis already in the directory
-            os.system('dcm2niix -b y -ba y -f ''"%d"'' -p y -z y -v 0 ''"'+acquisition+'"'' ')
+            os.system('dcm2niix -w 0 -b y -ba y -f ''"%d"'' -p y -z y -v 0 ''"'+acquisition+'"'' ')
```

Comment:
This was for running locally, the -w flag defines the write behavior for name conflicts (0,1,2, default 2: 0=skip duplicates, 1=overwrite, 2=add suffix). So when re-running a study locally it prohibited duplicating acquisitions. I've removed since we actually want the default option (2) when running across studies.